docs/blockchain-development-tutorials/forte/passkeys/index.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Passkeys (WebAuthn)
+title: Passkeys 
 description: Implement passkeys on Flow using WebAuthn, covering key extraction, challenges, signature formatting for Flow, and signature extensions.
 sidebar_position: 5
 keywords:
   - passkeys
 ---
 
-# Passkeys (WebAuthn)
+# Passkeys
 
-This is a wallet‑centric guide (per [FLIP 264: WebAuthn Credential Support]) that covers end‑to‑end WebAuthn integration for Flow:
+This is a wallet‑centric guide (per [FLIP 264: WebAuthn Credential Support]) that covers end‑to‑end passkeys integration for Flow:
 
-1. Register a passkey and add a Flow account key
+1. Create a passkey and add a Flow account key
```

Comment:
The standard uses "create" to generate the key and "register" to store it on the server (in our case store it on-chain)